documents/charter.md

```diff
@@ -1,23 +1,24 @@
+
+This is similar to what networks have seen in the past, such as in the 2000s when peer-to-peer (P2P) traffic posed a similar challenge and network operators were rate-shaping P2P traffic. In the United States, this practice was found to be counter to Net Neutrality (NN) [https://docs.fcc.gov/public/attachments/FCC-08-183A1.pdf]. Around this time the IETF held a workshop on Peer-to-Peer (P2P) Infrastructure, [https://www.rfc-editor.org/rfc/rfc5594.html]. Since that time, in the United States, wireline network operators have been barred from performing application-specific rate shaping, while wireless network operators have been permitted to do so. There regulator in the US, the Federal Communications Commission (FCC), currently has a proceeding underway, "Safeguarding and Securing the Open Internet", that may change this differential network management regime some time in 2024. 
 
 Video traffic usually employs adaptive bitrate (ABR) schemes to dynamically adjust the video quality (and thus the data rate) in response to changing network conditions.
-In the presence of traffic shaping, the ABR scheme should ideally adapt the quality and converge on a bitrate sustainable by the shaper.
+In the presence of a network operator performing traffic shaping (also known as throttling), the ABR scheme should ideally adapt the quality and converge on a bitrate sustainable by the shaper.
 In practice this is extremely difficult to achieve while maintaining a good user experience.
 Application providers are even designing algorithms to detect the presence of such traffic shapers and estimate the targeted shaping rate, however, these algorithms are likely to be both inaccurate and complex.
 Instead, it would be beneficial, for both the application provider and network operator, to signal the shaper rate to the application to self-adapt their video traffic to conform to the specified characteristics.
-The application provider has the ability to measure end user QoE and therefore can self-adapt with QoE feedback.
+The application provider has the ability to measure end user QoE and therefore can self-adapt with QoE feedback. Alternatively, networks could also cease traffic shaping of applications like ABR video streaming and, in addition, could further invest in network capacity. A final alternative is that application developers could detect that traffic was traversing a mobile network link and request a lower bitrate stream from application servers. 
```

Comment:
"traversing a mobile network link and request a lower bitrate stream from application servers." -> that's not always the case, per comments above on cellular context.

Reply:
Why isn't it the case that if a mobile app sees the active network interface is cellular that it applies its own shaping rule in the client? Or that a destination server or router can apply different rate shaping rules based on source ASN or source FQDN?

Reply:
I agree you would want to account for networks where higher bitrates may be a challenge - and it makes sense for the app to know it's on cellular where adverse/volatile conditions can cause that challenge - I just mean cellular may be able to support the higher bitrate (e.g. a strong 5G signal) so traffic shaping need not be triggered simply because the interface is cellular.